packages/client/components/DiscussionThreadListEmptyState.tsx

```diff
@@ -53,18 +49,14 @@ const DiscussionThreadListEmptyState = (props: Props) => {
   const message = `Start the conversation${
     allowTasks ? ' or add takeaway task cards' : ''
   } to capture next steps.`
+  const src = EmptyDiscussionIllustration
 
   return (
     <DiscussionThreadEmptyStateRoot>
       <EmptyDiscussionContainer>
-        <EmptyDiscussionSVG />
+        <EmptyDiscussionImage src={src} />
       </EmptyDiscussionContainer>
       <Message>{isReadOnly ? readOnlyMessage : message}</Message>
-      {!isReadOnly && (
-        <TangledArrowContainer>
-          <TangledArrowSVG />
```

Comment:
+1 I quite like the tangled arrow as it directs the user's eyes towards the search input. Do you feel that it doesn't work with this image?

Reply:
Thanks for the review @nickoferrall 🙌

Yep, the visual style of the arrow is different, and because the new illustration has much more detail than the previous one, I felt it was now a bit too heavy.